src/js/components/Text/Text.js

```diff
@@ -51,7 +51,11 @@ const Text = forwardRef(
 
     if (tip) {
       if (typeof tip === 'string') {
-        return <Tip content={tip}>{styledTextResult}</Tip>;
+        return (
+          <Tip content={tip} dropProps={rest?.dropProps}>
```

Comment:
Should we pull out `dropProps` as a documented prop where we mentioned it only take affect when `tip` is applied? and then just have this as  `dropProps={dropProps}`?

Reply:
Correct me if I am wrong, you meant to say that, we should pull out `dropProps` separately rather than using it as `rest?.dropProps` ?

Reply:
Yes, I think that is what @taysea is saying. We should add the prop on Text and use it directly instead of `rest?.dropProps`. Also we will need to add the prop to the documentation on [grommet-site](https://github.com/grommet/grommet-site) and specify that it should only be used when there is a `tip`.